docs/tutorials/AD-EdgeAI.ipynb

```diff
@@ -0,0 +1,2379 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "74600d4c",
+   "metadata": {},
+   "source": [
+    "# Edge AI Anomaly Detection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f536ae1",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "This document contains the code and the instructions for our EclipseCON 2022 Talk: \"_How to Train Your Dragon and Its Friends: AI on the Edge with Eclipse Kura&trade;_\"\n",
+    "\n",
+    "This notebook can also be viewed and ran on [Google Colab](https://colab.research.google.com/github/mattdibi/eclipsecon-edgeAI-talk/blob/master/notebook/AD-EdgeAI.ipynb).\n",
```

Comment:
Do you plan to keep this link alive?

Reply:
Yup. It's just a wrapper around the link to my repository: as long as the repo is up the Colab link will work.